packages/lib/src/vis/shared/SelectionMesh.tsx

```diff
@@ -28,13 +39,17 @@ function SelectionMesh(props: Props) {
       const { target, pointerId } = sourceEvent;
       (target as Element).setPointerCapture(pointerId);
 
+      if (!isEventValid(sourceEvent, modifierKey)) {
+        return;
+      }
+
       const point = worldToData(unprojectedPoint);
 
       setStartPoint(point);
       setEndPoint(point);
       setDrag(true);
```

Comment:
Random thought: maybe this should be called `setPanning` (and `isPanning`)?

Reply:
I wanted to convey the idea that we are _dragging_ the pointer to draw the shape. In my opinion, `isPanning` makes think of the pan from the `PanZoomMesh`.

Reply:
Oh sorry, mixed up the components, thought this was in `PanZoomMesh` 😅